src/current/_includes/v23.2/storage/compaction-concurrency.md

```diff
@@ -0,0 +1 @@
+By default, the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) uses a compaction concurrency of 3. If you have sufficient IOPs and CPU headroom, you can consider increasing this setting via the `COCKROACH_COMPACTION_CONCURRENCY` environment variable. This may be useful in the following scenarios: it may help to reshape the LSM faster in [inverted LSM]({% link {{ page.version.version }}/architecture/storage-layer.md %}#inverted-lsms) scenarios; and it can lead to increased overall performance for some workloads. We **strongly recommend** testing your workload against non-default values of this setting.
```

Comment:
```suggestion
By default, the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) uses a compaction concurrency of 3. If you have sufficient IOPS and CPU headroom, you can consider increasing this setting via the `COCKROACH_COMPACTION_CONCURRENCY` environment variable. This may help to reshape the LSM more quickly in [inverted LSM]({% link {{ page.version.version }}/architecture/storage-layer.md %}#inverted-lsms) scenarios; and it can lead to increased overall performance for some workloads. Cockroach Labs **strongly recommends** testing your workload against non-default values of this setting.
```